src/components/Home.vue

```diff
@@ -1,31 +1,21 @@
 <script setup>
-import { ref, defineProps, watchEffect } from "vue";
-import UserContributionForm from "./UserContributionForm.vue";
+// import { stringify } from "querystring";
```

Comment:
Let's not merge commented out code, we can delete it now and add it back when we need it.

Reply:
Addressed in [b3895cc](https://github.com/wikimedia/toolhunt-ui/pull/41/commits/b3895ccc14df0fc0d4bc2bec6257fa73d9fa4ecb)